### DISCUSSION TITLE
Is it possible to add actions to notification in foreground mode?

### DISCUSSION THREAD

**Discussion by AbdullahSohail-SE** [HaylLtd/cordova-plugin-background-geolocation#145] [Q&A]
Consider an example below, the app has an open and stop button, the stop button stops the tracking service, is it possible to have such interaction via this plugin? can we add actions to the notifications panel somehow?
![Screenshot_2023-04-12-17-43-52-34](https://user-images.githubusercontent.com/51018513/231461176-2f0e33c9-0062-47e5-bf7e-1b6d1ff37cce.jpg)

**Comment by HarelM** [comment]
Everything is possible, I would recommend forking, experimenting and if everything works as expected, open a PR. From my point of view, stopping tracking from "outside the app" is problematic, but I'll be interested to know how one can solve these issues.
